docs/en/observability/synthetics-configuration.asciidoc

```diff
@@ -77,6 +77,11 @@ Read more in <<synthetics-params-secrets>>.
 
 For all available options, refer to the https://playwright.dev/docs/test-configuration[Playwright documentation].
 
+[NOTE]
+====
+Synthetics will always run browser monitors in headless mode (it is not possible to configure them to run in headful mode, even if setting Playwright options).
```

Comment:
I would have said "even forcing `headless:false` via Playwright options, we do not support running browser monitors in headful mode".

+ possibly we might give an hint of passing a user agent like here if really needed? https://github.com/elastic/synthetics/blob/d33d899bafa686da0b547afd2145db51b411ed1b/examples/todos/synthetics.config.ts#L12

Reply:
@lucabelluccini I thought the confusion was originally because someone was specifically trying to run in headful mode - were they doing this because they wanted a non headless UA string then?

Reply:
I do not have the whole detail about it, we can check together in Slack